snap-build/xbuild.sh

```diff
@@ -0,0 +1,161 @@
+#!/bin/bash
+#
+# Copyright (c) 2018 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+# Build in parallel snap images using VMs.
+# This script runs in the host.
+
+set -x -e
+
+source lib.sh
+
+readonly supported_archs=(all amd64 ppc64 arm64)
```

Comment:
Should this be `arm` to match the change in `.ci/lib.sh`?

Reply:
nop, arm64 is correct, the change in `.ci/lib.sh` is to match with `yq` releases